test_istgt.sh

```diff
@@ -570,14 +570,28 @@ check_healthy_replica_count()
 {
 
 	no_of_replicas=$1
+	local i=0
 
 	cmd="$ISTGTCONTROL -q REPLICA vol1 | jq '.\"volumeStatus\"[0].\"replicaStatus\"[].Mode' | grep -w Healthy"
 	cnt=$(eval $cmd | wc -l)
 
-	if [ $cnt -ne $no_of_replicas ]
+	## If desired count is not met, checking for desired count for 5 minutes with an interval of 3 seconds.
+	if [ $cnt -ne $no_replicas ]
```

Comment:
here, is this supposed to be no_of_replicas? also, one more similar one at next few lines.

Reply:
Yes it is supposed to be `no_of_replicas` I will update it.